### PR TITLE
Generates core weekly for weeks 37, 38 and 39

### PR DESCRIPTION
**Pull request by matks**
2nd batch of automated core weekly posts

### CODE REVIEW

File: news/_posts/core-weekly/2019-09-29-coreweekly-week-39-2019.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 23th of September to Sunday 29th of September 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+[...]
```
Comment (matks) [inline news/_posts/core-weekly/2019-09-29-coreweekly-week-39-2019.md]: This is the core weekly for last week, I think we can add some content. Maybe we can talk about the new core weekly generator ?

Comment (ttoine) [reply news/_posts/core-weekly/2019-09-29-coreweekly-week-39-2019.md]: yes, good idea !

**Comment by ttoine** [comment]
LGTM

**Comment by matks** [comment]
- Need to apply this fix https://github.com/PrestaShop/prestashop.github.io/issues/583
- Need to update dates
- Need to add content to the last core weekly

**Comment by matks** [comment]
Posts ready to be published tomorrow, 11am Paris time

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**